docs/website/docs/contributing.md

```diff
@@ -164,35 +164,35 @@ In order to achieve this:
 
 :::info[Walkthrough]
 
-Let's imagine that we need to add a dependency to the `marked` package to `@bonfhir/react`.
+Let's imagine that we need to add the `remarkable` package as a dependency of `@bonfhir/react`.
```

Comment:
I changed the documentation here to give the example with `remarkable` since `marked` is no longer a part of the project anyways. This isn't required but felt relevant.